### ISSUE TITLE
Announcement emails sent blank after upgrade

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#6084]
**Describe the bug**
From https://forum.pkp.sfu.ca/t/error-with-announcement-notification-emails/61663:
> We have recently upgraded from ojs3.2.1 to 3.2.1-1
We have generated an announcement for our readers but it sent empty emails to the journal readers. I tried it once more, but it again created an empty email without any subject and content in it. I got many emails from the readers about sending empty emails. I cannot generate any new announcement out of fear of again sending empty emails. It was working fine before the update. Is there any solution for it and can we test it without sending the email notification?

**To Reproduce**
1. Start with OJS 3.x < 3.2.1
2. Upgrade to OJS 3.2.1
3. Try sending an announcement
4. Emails will be received blank.

**What application are you using?**
 OJS, OMP or OPS version 3.2.1

This is caused by the new `ANNOUNCEMENT` template being added (https://github.com/pkp/pkp-lib/issues/4746), but not installed upon upgrade.

**Comment by asmecher** [comment]
(The code appears present to take care of the upgrade -- this is potentially just a case of missing translations.)

**Comment by asmecher** [comment]
The issue is likely caused by the upgrade script only installing the English (`en_US`) language template. The attached commits will add it upon upgrade in the other languages in which it's currently translated, and this will be included in the next release (3.2.1-3) for users who haven't upgraded to 3.2.1-x yet.

For users who have already upgraded to 3.2.1-x older than 3.2.1-2, there are several options:
- If you have access to the command line, use the command-line tool:
  - If you are using OJS 3.2.1-0 or 3.2.1-1, you can [get the tool from here](https://raw.githubusercontent.com/pkp/pkp-lib/stable-3_2_1/tools/installEmailTemplate.php) and save it to `lib/pkp/tools/installEmailTemplate.php`.
  - Run it like so:
    ```
    cd /path/to/ojs
    php lib/pkp/tools/installEmailTemplate.php ANNOUNCEMENT ca_ES,cs_CZ,ar_IQ,da_DK,es_ES,fr_CA,id_ID,it_IT,pt_BR,pt_PT,ru_RU,sl_SI,sv_SE,tr_TR,vi_VN,zh_CN
    ```
    ...customizing the list of locales to the ones you need.

- Alternately, *IF YOU HAVE NOT CUSTOMIZED YOUR EMAIL TEMPLATES*, you can add the translations using the web interface. (This will delete customized email template wording for all journals for the languages you choose!)
  - Log in as site administrator,
  - Navigate to Settings > Website > Languages,
  - For each language you intend to send this template with, use the "Reload defaults" tool.
- Alternately, if you have modified your email templates or do not wish to use the above instructions:
  - For each journal you need to use this template with:
    - Log in as journal manager
    - Navigate to Settings > Workfow > Emails > Email Templates
    - Edit the email template with the description `This email is sent when a new announcement is created.`
    - For each language you wish to send this template with, add translations. You can get the default wording from [GitHub](https://github.com/pkp/ojs/blob/stable-3_2_1/locale/en_US/emails.po#L14..L28), replacing `en_US` with the name of the language you want to see. Available language codes are visible [here](https://github.com/pkp/ojs/tree/stable-3_2_1/locale).
